test/evergreen.yml

```diff
@@ -1990,15 +1990,16 @@ tasks:
           # run for 24 hours ( 24 * 60 = 1440 minutes), skip known errors, don't stop at failed tests, use default config
           format_test_script_args: -E -t 1440
 
-  - name: format-stress-sanitizer-lsm-test
-    commands:
-      - func: "get project"
-      - func: "compile wiredtiger address sanitizer"
-      - func: "format test script"
-        vars:
-          test_env_vars: ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0" ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer
-          # Run for 30 mins, and explicitly set data_source to LSM with a large cache
-          format_test_script_args: -t 30 data_source=lsm cache_minimum=5000
+  # Temporarily disabled (WT-6255)
+  # - name: format-stress-sanitizer-lsm-test
+  #   commands:
+  #     - func: "get project"
+  #     - func: "compile wiredtiger address sanitizer"
+  #     - func: "format test script"
+  #       vars:
+  #         test_env_vars: ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0" ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer
+  #         # Run for 30 mins, and explicitly set data_source to LSM with a large cache
+  #         format_test_script_args: -t 30 data_source=lsm cache_minimum=5000
```

Comment:
This part of change should be optional.

Reply:
Good point! I updated to just comment out the task line.